lib/views/items_details.erb

```diff
@@ -60,16 +60,40 @@
           <form method="POST" action="/items-details">
             <div class="form-row">
               <div style="width:14%;" class="p-1 pl-3">  
+                <% if @errors.include?(:missing_id)%>
+                  <input type="text" id="id" name="id" class="form-control is-invalid" placeholder="ID"/>
+                <% elsif @item_row[:id].nil? %>
                 <input type="text" id="id" name="id" class="form-control" placeholder="ID"/>
+                <% else %>
+                  <input type="text" id="id" name="id" class="form-control" value="<%= @item_row[:id]%>"/>
+                <% end %>
               </div>
               <div style="width:45%;" class="p-1">  
-                <input type="text" id="name" name="name" class="form-control" placeholder="Name"/>
+                <%if @errors.include?(:missing_name)%>
+              <input type="text" id="name" name="name" class="form-control is-invalid" placeholder="Name"/>
+                <% elsif @item_row[:name].nil? %>
+                  <input type="text" id="name" name="name" class="form-control" placeholder="Name"/>
+              <% else %>
+                <input type="text" id="name" name="name" class="form-control" value="<%= @item_row[:name]%>"/>
+              <% end %>
               </div>
               <div style="width:10%;" class="p-1">
+                <% if @errors.include?(:missing_price) || @errors.include?(:invalid_price)%>
+              <input type="text" id="price" name="price" class="form-control is-invalid" placeholder="Price"/>
+              <% elsif @item_row[:price].nil? %>
                 <input type="text" id="price" name="price" class="form-control" placeholder="Price"/>
+              <% else %>
+                <input type="text" id="price" name="price" class="form-control" value="<%= @item_row[:price]%>"/>
+              <% end %>
               </div>
               <div style="width:11%;" class="p-1">
+                <%if @errors.include?(:missing_quantity) || @errors.include?(:invalid_quantity)%>
```

Comment:
Should we replace this line with `<%if @errors.include?(:missing_quantity || :invalid_quantity) %>` ?